pingdom/api_responses.go

```diff
@@ -19,26 +19,33 @@ type PingdomError struct {
-	SendToEmail              bool              `json:"sendtoemail,omitempty"`
-	SendToIPhone             bool              `json:"sendtoiphone,omitempty"`
-	SendToSms                bool              `json:"sendtosms,omitempty"`
-	SendToTwitter            bool              `json:"sendtotwitter,omitempty"`
-	SendNotificationWhenDown int               `json:"sendnotificationwhendown,omitempty"`
-	NotifyAgainEvery         int               `json:"notifyagainevery,omitempty"`
-	NotifyWhenBackup         bool              `json:"notifywhenbackup,omitempty"`
-	Created                  int64             `json:"created,omitempty"`
-	Hostname                 string            `json:"hostname,omitempty"`
-	Status                   string            `json:"status,omitempty"`
-	LastErrorTime            int64             `json:"lasterrortime,omitempty"`
-	LastTestTime             int64             `json:"lasttesttime,omitempty"`
-	LastResponseTime         int64             `json:"lastresponsetime,omitempty"`
-	Paused                   bool              `json:"paused,omitempty"`
-	ContactIds               []int             `json:"contactids,omitempty"`
-	Type                     CheckResponseType `json:"type,omitempty"`
+	ID                       int                `json:"id"`
+	Name                     string             `json:"name"`
+	Resolution               int                `json:"resolution,omitempty"`
+	SendToAndroid            bool               `json:"sendtoandroid,omitempty"`
+	SendToEmail              bool               `json:"sendtoemail,omitempty"`
+	SendToIPhone             bool               `json:"sendtoiphone,omitempty"`
+	SendToSms                bool               `json:"sendtosms,omitempty"`
+	SendToTwitter            bool               `json:"sendtotwitter,omitempty"`
+	SendNotificationWhenDown int                `json:"sendnotificationwhendown,omitempty"`
+	NotifyAgainEvery         int                `json:"notifyagainevery,omitempty"`
+	NotifyWhenBackup         bool               `json:"notifywhenbackup,omitempty"`
+	Created                  int64              `json:"created,omitempty"`
+	Hostname                 string             `json:"hostname,omitempty"`
+	Status                   string             `json:"status,omitempty"`
+	LastErrorTime            int64              `json:"lasterrortime,omitempty"`
+	LastTestTime             int64              `json:"lasttesttime,omitempty"`
+	LastResponseTime         int64              `json:"lastresponsetime,omitempty"`
+	Paused                   bool               `json:"paused,omitempty"`
+	ContactIds               []int              `json:"contactids,omitempty"`
+	Type                     CheckResponseType  `json:"type,omitempty"`
+	Tags                     []CheckResponseTag `json:"tags,omitempty"`
+}
+
+type CheckResponseTag struct {
+	Name  string `json:"name"`
+	Type  string `json:"type"`
+	Count int    `json:"count"`
```

Comment:
this actually has to be interface{} sometime count is retruened as a string.

Reply:
According to the documentation, on might think that the API returns an int but, in fact, it returns a string.
Maybe we could use a type string for this field?

Reply:
Just used that and it return `int` for me for api/2.1